packages/angular/projects/angular/src/compatible-components/radio-group/radio-group-compatible.component.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  ChangeDetectionStrategy, Component, ContentChild, Input,
+} from '@angular/core';
+import { RadioGroupValue } from '@devextreme/components/src';
+import { compileGetter, ItemLike } from '@devextreme/interim';
+import { TemplateCompatibleDirective } from '../../compatible-directives/template';
+import { RadioGroupBaseComponent } from '../../components/radio-common';
+
+// TODO: Move these to components package
```

Comment:
I especially don't mention myself in this TODO.
Because if we start moving these common types from React it can be done in Angular in the same PR.
And this task may be assigned on another developer :)